doc/man3/EVP_PKEY_keygen.pod

```diff
@@ -138,10 +138,11 @@ Generate a key from a set of parameters:
  #include <openssl/rsa.h>
 
  EVP_PKEY_CTX *ctx;
+ ENGINE *eng;
  EVP_PKEY *pkey = NULL, *param;
 
- /* Assumed param is set up already */
- ctx = EVP_PKEY_CTX_new(param);
+ /* Assumed eng, param are set up already */
```

Comment:
But here it's `assumed`. I suppose what stings the eye is lack of subject in sentence...

Reply:
I am not sure from this comment what is your preference.

Reply:
Well, if "assumes" sounds right, then I'd say that this one should also read "assumes". What I meant was that they were *different*, they *both* stung my eye.

Reply:
Well ... if you grep through the all source code, you will get both occurrences with very similar amount:
```
$ git grep -i assumes | wc -l
58
$ git grep -i assumed | wc -l
54
```
so I would not bother with wording in this PR, but if you think, it is worth fixing it to the same wording, it is probably idea for a next PR/commit.